docs/guides/managing-existing-services.md

```diff
@@ -93,14 +94,42 @@ Harmony validates that the image and tag are reachable - an error will be return
 **Important** from the [Docker documentation](https://docs.docker.com/engine/reference/commandline/image_tag/):
 >A tag name may contain lowercase and uppercase characters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.
 
+## Get backend service image tag update status
+
+You can get the status of backend service tag update by following the `status_link` returned in the backend service tag update response.
+
+For example:
+
+```
+curl -XGET -Ln -bj https://harmony.uat.earthdat.nasa.gov/service-image-tag/deployment/<deployment-id>
+```
+**Example 8** - Get service image tag update status
+
+The returned JSON response has the fields indicating the current status of the service image deployment.
+
+```JSON
+{
+  id: 1,
+  deployment_id: "befb50e0-e467-4776-86c8-e7218f1123cc",
```

Comment:
I noticed both deployment_id and status_link are snake case, but other things like createdAt and updatedAt are camel case. On the jobs API we use camel case for everything so maybe use camel case everywhere for consistency?

Reply:
I thought about doing it, but decided not to because the camel cases are from the db schema, given that we are changing all db column names to snake case, we probably want to get all the fields to snake case as well in the future. Looks like it bothers you, I can change it to camel case for now and we can change it back to snake case when we eventually change the db schemas.

Reply:
The API and the database fields should be treated separately. We shouldn't have the API try to match the database, and we may not necessarily want to include all database fields in API responses.